mbedtls/tests/ssl_conf_psk_cb.rs

```diff
@@ -0,0 +1,61 @@
+#![allow(dead_code)]
```

Comment:
Shouldn't be needed

Reply:
Removed

Reply:
See update in: https://github.com/fortanix/rust-mbedtls/pull/205#issuecomment-1205973141